### PR TITLE
chore: upgrade flutter to 3.19.1

### LINKED ISSUE #1924: Upgrade Flutter

### PR DESCRIPTION
**Pull request by bonomat**
resolves #1924 

Note: 
1. The overall design changed slightly, I think whenever working on the UI we need to adjust minor things... or live with it that the color tone changed a bit.
2. this does not upgrade flutter rust bridge which is a beast in itself

### CODE REVIEW

**Comment by bonomat** [comment]
@holzeis : fixed the firefox issue in [`e43730f` (#2081)](https://github.com/get10101/10101/pull/2081/commits/e43730ff03725085d13201bcbe1025a096fede18)

**Comment by holzeis** [comment]
Two more things.

- The payout chart doesn't fit anymore, throwing a layout error.

```
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
The following assertion was thrown during layout:
A RenderFlex overflowed by 40 pixels on the bottom.

The relevant error-causing widget was:
  Column
  Column:file:///Users/holzeis/workspace/coblox/10101/mobile/lib/features/welcome/payout_chart.dart:79:12

To inspect this widget in Flutter DevTools, visit:
http://127.0.0.1:9100/#/inspector?uri=http%3A%2F%2F127.0.0.1%3A56088%2FtVP8iz4aw_0%3D%2F&inspectorRef=inspector-0

The overflowing RenderFlex has an orientation of Axis.vertical.
The edge of the RenderFlex that is overflowing has been marked in the rendering with a yellow and
black striped pattern. This is usually caused by the contents being too big for the RenderFlex.
Consider applying a flex factor (e.g. using an Expanded widget) to force the children of the
RenderFlex to fit within the available space instead of being sized to their natural size.
This is considered an error condition because it indicates that there is content that cannot be
seen. If the content is legitimately bigger than the available space, consider clipping it with a
ClipRect widget before putting it in the flex, or using a scrollable container rather than a Flex,
like a ListView.
The specific RenderFlex in question is: RenderFlex#bded4 relayoutBoundary=up6 OVERFLOWING:
  creator: Column ← PnlLineChart ← FractionallySizedBox ← Expanded ← Column ← Expanded ← Column ←
    ConstrainedBox ← Container ← Transform ← Center ← AnimatedBuilder ← ⋯
  parentData: offset=Offset(0.0, 23.5) (can use size)
  constraints: BoxConstraints(0.0<=w<=390.0, h=188.2)
  size: Size(390.0, 188.2)
  direction: vertical
  mainAxisAlignment: spaceEvenly
  mainAxisSize: max
  crossAxisAlignment: center
  verticalDirection: down
◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤
════════════════════════════════════════════════════════════════════════════════════════════════════
```

- The channel config modal is not moved when the keyboard is toggled.

Simulator Screenshot - iPhone 13 Pro - 2024-02-24 at 10 10 56

**Comment by bonomat** [comment]
@holzeis : 

> Two more things.
> 
>     * The payout chart doesn't fit anymore, throwing a layout error.

I'm giving up on this :/ Might be better to spend some time replacing the carousel but I don't have the nerves for this right now. 

> ```
> 
>     * The channel config modal is not moved when the keyboard is toggled.

This is fixed: 

image

**Comment by holzeis** [comment]
> I'm giving up on this :/ Might be better to spend some time replacing the carousel but I don't have the nerves for this right now.

Maybe this gets "fixed" if we switch back to the previous font and size.

**Comment by bonomat** [comment]
> Maybe this gets "fixed" if we switch back to the previous font and size.

Be my guest figuring out what it was before and setting it to this value now 🙈

**Comment by holzeis** [comment]
The small dots indicating that there is a carousel are also gone.

Screenshot 2024-02-26 at 09 57 07

File: webapp/frontend/lib/auth/login_screen.dart [hunk 1]
```diff
@@ -31,12 +31,11 @@ class _LoginScreenState extends State<LoginScreen> {
         Image.asset('assets/10101_logo_icon.png', width: 350, height: 350),
         SizedBox(
             width: 500,
-            height: 150,
             child: Container(
               padding: const EdgeInsets.all(18),
               decoration: BoxDecoration(
                 borderRadius: BorderRadius.circular(8),
-                color: Colors.grey[100],
+                // color: Colors.grey[100],
```
Comment (luckysori) [inline webapp/frontend/lib/auth/login_screen.dart:38]: 🔧 I guess we can remove this.

**Review by luckysori — APPROVED** [review]
LGTM just looking at the diff.

Although I was confused because I thought we were tackling https://github.com/get10101/10101/issues/1777 haha.

**Pull request merged**